src/fragments/lib/push-notifications/common/getting-started.mdx

```diff
@@ -25,13 +29,21 @@ import ios9 from "/src/fragments/lib/push-notifications/ios/getting_started/40_c
 
 <Fragments fragments={{ios: ios9}} />
 
+import android4 from "/src/fragments/lib/push-notifications/android/getting_started/20_cli_resources.mdx";
+
+<Fragments fragments={{android: android4}} />
+
 </Block>
 <Block name="Existing resources">
```

Comment:
@abdallahshaban557 this feels a bit confusing to me, in other categories existing resources is a different tab on the sidebar, why is it different here?

Reply:
We are actually moving all of the "use existing resources" to be inside of the getting started pages. Developers think that using the Amplify libraries requires the CLI - however, that is incorrect. So we are trying to make that as clear as possible.

Reply:
Shouldn't we remove them all at once then for consistency?

Reply:
we have a PR out for Auth right now - and then Storage will follow - then API.